packages/server/__tests__/jiraServerIntegration.test.ts

```diff
@@ -51,7 +51,7 @@ test('Add integration provider', async () => {
         oAuth1ProviderMetadataInput: {
           serverBaseUrl,
           consumerKey,
-          privateKey
+          consumerSecret
```

Comment:
cc @Dschoordsch - this was broken upstream

Reply:
Yeah, that's me wo broke it

Reply:
No worries! Just making sure this seems correct to you.